ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.create-run.tsx

```diff
@@ -369,7 +374,30 @@ const CreateRunPage = () => {
                   <FormControl>
                     <Input {...field} />
                   </FormControl>
-                  <FormDescription>Revision to run ORT on</FormDescription>
+                  <FormDescription>
+                    The repository revision used by this run
+                  </FormDescription>
+                  <FormMessage />
+                </FormItem>
+              )}
+            />
+            <FormField
+              control={form.control}
+              name='jobConfigContext'
+              render={({ field }) => (
+                <FormItem className='pt-4'>
+                  <FormLabel>Job configuration context</FormLabel>
+                  <FormControl>
+                    <Input {...field} />
+                  </FormControl>
+                  <FormDescription>
+                    An optional context to be used when obtaining configuration
+                    for this ORT run. This context is passed to the
```

Comment:
I believe we should not make too much use of "ORT" in descriptions like this. To the user, it's an implementation detail that ORT is used underneath. So maybe just replace "ORT run" with "job".

Reply:
The next two sentences I'd probably drop, and instead of giving concrete examples, simply state generally like over [here](https://github.com/eclipse-apoapsis/ort-server/blob/fe810f368363e9a8ec4a4ab2604cb938a5bcf90e/config/spi/src/main/kotlin/Context.kt#L26) that "the meaning of the context is up for interpretation by the implementation of the configuration provider". (Or so.)